res/text.v.glsl

```diff
@@ -44,6 +51,11 @@ uniform int backgroundPass;
 
 void main()
 {
+    skipSpacer = isSpacer;
+    if (skipSpacer != 0) {
+        return;
+    }
```

Comment:
I don't think it makes sense to send data to the shader, just to always discard it?

Reply:
Are you saying we should just remove the spacer rendering operation from the Rust code?
Let me have a try.

Reply:
Yes. We are already filtering out whitespace characters. So if the spacer is not needed for rendering, we should filter it out. Though make sure to filter both the leading and trailing spacers.